src/org.openSUSE.YaST.appdata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id>org.openSUSE.YaST</id>
+  <launchable type="desktop-id">org.openSUSE.YaST.desktop</launchable>
+  <metadata_license>GPL-2.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>YaST</name>
+  <summary>Configuration and System Management Software</summary>
+  <description>
+    <p>YaST is both an extremely flexible installer and a powerful control center. It's an all-purpose tool for system management.</p>
+    <p>A multitude of modules allows to flexibly extend the capabilities of YaST.</p>
+  </description>
+  <categories>
+    <category>Settings</category>
+    <category>HardwareSettings</category>
+    <category>DesktopSettings</category>
+  </categories>
+  <screenshots>
+    <screenshot type="default">https://yast.opensuse.org/assets/images/screenshots/screenshot_1.png</screenshot>
```

Comment:
Um, the https://yast.opensuse.org addresses report a certificate issue, maybe we should use a plain `http://` addresses until that's fixed...

Reply:
Eh, forgot about that